opensubtitles/download.go

```diff
@@ -0,0 +1,60 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+type DownloadService service
```

Comment:
Codacy found an issue: [exported type DownloadService should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)